paper-dropdown-menu-light.html

```diff
@@ -139,9 +139,7 @@
         @apply --paper-dropdown-menu-input;
       }
 
-      /* Keep :host-context() to make it work where :dir doesn't exist */
-      :host(:dir(rtl)) #input,
-      :host-context([dir="rtl"]) #input {
+      #input:dir(rtl) {
```

Comment:
shouldn't this be `:host(:dir(rtl)) #input` ?

Reply:
`:host` would be superfluous in this case, and as suggested in [this comment](https://github.com/Polymer/polymer/pull/4964#discussion_r153965363) what we want in this case is the compound selector which would handle both the case of `<input dir="rtl">` and `<html dir="rtl"> ..... <input>`

Reply:
but the input is in _your_ shadow dom, so who's going to set `dir="rtl"` on it?

Reply:
true, no-one will set it on the input. The compound selector tho will have lower specificity and  clearer to read. The end result is the same, right?

Reply:
Talked to @azakus: I understand now.